services/tracing/tracing/transport/http.go

```diff
@@ -0,0 +1,71 @@
+package tracingtransport
+
+import (
+	"fmt"
+	"log"
+	"net/http"
+
+	"github.com/go-chi/chi/v5"
+	"sensorbucket.nl/sensorbucket/internal/httpfilter"
+	"sensorbucket.nl/sensorbucket/internal/pagination"
+	"sensorbucket.nl/sensorbucket/internal/web"
+	"sensorbucket.nl/sensorbucket/services/tracing/tracing"
+)
+
+type HTTPTransport struct {
+	router chi.Router
+	svc    *tracing.Service
+	url    string
+}
+
+func NewHTTP(svc *tracing.Service, url string) *HTTPTransport {
+	t := &HTTPTransport{
+		router: chi.NewRouter(),
+		svc:    svc,
+		url:    url,
+	}
+	t.SetupRoutes(t.router)
+	return t
+}
+
+func (t *HTTPTransport) SetupRoutes(r chi.Router) {
+	r.Get("/health", func(rw http.ResponseWriter, r *http.Request) {
+		rw.Write([]byte("healthy"))
+	})
```

Comment:
Not required as it does not checky any health related things

Reply:
removed